omero/sysadmins/customization.txt

```diff
@@ -6,7 +6,7 @@ The sections below describe how to set up these features.
 
 Note that depending on the deployment choice, OMERO.web will not activate
 configuration changes until either gunicorn is restarted using ``bin/omero web
-restart`` (nginx) or the Apache webserver has been reloaded.
+restart`` (nginx).
```

Comment:
Remove 'either' in previous line. Maybe also remove ```(nginx)``` since I'm not sure it's needed now?